photon-api/src/main/scala/com/linkedin/photon/ml/util/MathUtils.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2016 LinkedIn Corp. All rights reserved.
```

Comment:
Should we update those copyright notices to include 2017? The copyright might be void if we leave it as 2016, but I'm not a lawyer.